include/zephyr/drivers/swdp.h

```diff
@@ -18,28 +18,76 @@
 extern "C" {
 #endif
 
-/* SWDP packet request bits */
+/**
+ * @defgroup swdp_interface SWDP Interface
+ * @since 3.7
+ * @version 0.1.0
+ * @ingroup io_interfaces
+ * @{
+ */
+
+/**
+ * @name SWDP Packet Request Bits
```

Comment:
Likely I messed up naming, it should be just SWD Packet ` * @name SWD Packet Request Bits`

Reply:
Ah, makes sense indeed. I believe I have applied all the comments